src/Tribe/Admin/Bar/Default_Configurator.php

```diff
@@ -15,7 +15,7 @@ public function configure( WP_Admin_Bar $wp_admin_bar ) {
 
 		$wp_admin_bar->add_menu( array(
 			'id'    => 'tribe-events',
-			'title' => '<span class="ab-icon dashicons-before dashicons-calendar"></span>' . sprintf( __( '%s', 'the-events-calendar' ), $main->plural_event_label ),
+			'title' => '<span class="ab-icon dashicons-before dashicons-calendar"></span>' . sprintf( _x( '%s', 'Plural form of the word "Event"', 'the-events-calendar' ), $main->plural_event_label ),
```

Comment:
Is this a good opportunity to use `$this->taxonomyLabels['name']` instead?
https://developer.wordpress.org/reference/functions/get_post_type_labels/

I found 9 other uses of `->plural_event_label` within TEC (didn't check other plugins)

===

If leaving it as-is, maybe `esc_html_x()` instead